### PR TITLE
Tip to install email-validator package to use email validation

### PR DESCRIPTION
**Pull request by manlix**
Fix for issue in documentation example code #3368

### CODE REVIEW

File: docs/en/docs/tutorial/response-model.md [hunk 1]
```diff
@@ -33,6 +33,11 @@ But most importantly:
 
 ## Return the same input data
 
+!!! info
+    To use email validator (`EmailStr` type), first install <a href="https://github.com/JoshData/python-email-validator" class="external-link" target="_blank">`email-validator`</a> package.
+
+    E.g. `pip install email-validator`.
+
```
Comment (Bluenix2) [inline docs/en/docs/tutorial/response-model.md:40]: Personally I rather see this as a note under the code worded more like:
```suggestion
!!! note
    `EmailStr` uses <a href="https://github.com/JoshData/python-email-validator" class="external-link" target="_blank">`email-validator`</a>, you will have to install this package to use `EmailStr`.

```

**Review by Bluenix2 — CHANGES_REQUESTED** [review]

**Review by Bluenix2 — CHANGES_REQUESTED** [review]
This is misleading, now the user is let on to belive that to use `response_model` you need to install `email-validator`.

**Comment by vrslev** [comment]
I don’t think the instructions belong here. Pydantic uses `email-validator`. Imagine what it would be like if FastAPI would follow and add all remarks from its dependencies 😃

**Comment by tiangolo** [comment]
Thank you! This was solved recently in another PR.

Given that, I'll close this one now. But thanks for the effort! :cake:

**Pull request closed without merging**